### DISCUSSION TITLE
PointcloudFromArray Texture Coordinates not working ?

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#450] [Answers & Questions]
i tried setting the texture coordinates for each point via the TextureCoordinates array input but nothing happens. then i tried to "scramble" the texture coordinate with built in option... nothing happens. looking at the code of the op there is an empty tex coords array initialised, maybe that the reason ? or is this user error and i'm doing something wrong ?

it would help to see a tex coords example of pointcloudfromarray, where every point gets a different texcoord.

![image](https://user-images.githubusercontent.com/1138673/211017221-9c68b238-a4c3-4d74-9872-42327a03662e.png)

**Comment by steam0r** [comment]
could you provide an example patch for this behaviour please? might be related to the default checkboxes

**Comment by kdklv** [comment]
noticed similar thing in performance op
![image](https://user-images.githubusercontent.com/18003839/211289471-8b40a428-3ebf-48bc-8976-c63099f95ebd.png)
